utils/log.go

```diff
@@ -114,7 +124,12 @@ func NewZapLogger(logLevel LogLevel, colour bool) (*ZapLogger, error) {
 	if err != nil {
 		return nil, err
 	}
-	config.Level.SetLevel(level)
+	if logLevel == TRACE {
```

Comment:
Maybe something like?

	var level zapcore.Level
	var err error

	if logLevel.String() == TRACE.String() {
		level = LevelTrace
	} else {
		level, err = zapcore.ParseLevel(logLevel.String())
		if err != nil {
			return nil, err
		}
	}
	config.Level.SetLevel(level)

Reply:
Currently some tests will fail because 
 zapcore.ParseLevel(logLevel.String())
 returns an error when loglevel is trace